.github/workflows/ci-tests.yml

```diff
@@ -108,4 +112,7 @@ jobs:
       env:
         POST_COMMAND: ${{ matrix.post-command }}
       run: |
+        export PATH=${{ matrix.gitpath-prepend }}$PATH
+        which git
+        git --version
```

Comment:
Are we leaving these in? I suppose I could get behind that because this is a hack.